test/e2e/operator_test.go

```diff
@@ -19,10 +19,34 @@ import (
 
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
+const (
+	// upstreamPodName is the name of the upstream CoreDNS server
+	// used for testing the DNS forwarding feature.
+	upstreamPodName = "test-upstream"
+	// upstreamPodNS is the namespace of the upstream CoreDNS server
+	// used for testing the DNS forwarding feature.
+	upstreamPodNS = "openshift-dns"
+	// upstreamCorefile is the Corefile used by the upstream CoreDNS server
+	// used for testing the DNS forwarding feature.
+	upstreamCorefile = `.:5353 {
+    hosts {
+      1.2.3.4 www.foo.com
+    }
+    health
+    errors
+    log
+}
+`
+	// coreStatusVersion is the name of the CoreDNS status version for
+	// the dns clusteroperator.
+	coreStatusVersion = "coredns"
```

Comment:
Can you use `operatorcontroller.CoreDNSVersionName` instead of introducing a new const?

Reply:
ack